src/aleph/vm/models.py

```diff
@@ -161,15 +168,17 @@ async def prepare(self):
             if not resources:
                 msg = "Unknown executable message type"
                 raise ValueError(msg, repr(self.message))
-            await resources.download_all()
+            if download:
```

Comment:
Why add this `download` flag ? Is `download_all()` not checking for existing resources ? What would happen if a node operator clears `/var/lib/cache` before restarting the orchestrator ?

Reply:
Yes, it checks that already exists. Was the runtime that I was worried about, but it also checks if already exists. I will remove that argument.